tests/integration/report/test_report.py

```diff
@@ -62,6 +62,297 @@ def test_sidebar(report_data_fixture):
     )
 
 
+def test_duplicate_rows_for_latest_run_status(report_data_fixture):
+    report_data = get_report_data()
+    # e2e contains 2 failed dimension tests and 1 passed
+    assert_test_counter(
+        report_data=report_data,
+        test_type="anomaly_detection",
+        name="dimension_anomalies",
+        status="fail",
+        expected_ammount=2,
```

Comment:
Let's change the counter to be more around validating the test sub types (for each sub type we should validate exactly one row)